setup_venv.ps1

```diff
@@ -27,10 +27,10 @@
   None
 
 .NOTES
-  Version        1.0
-  Author         powderluv, xzuyn
-  Creation Date  2023-02-17
-  PurposeChange Initial script development
+  Version        1.01
```

Comment:
I'd like to avoid version numbers here and authors. Version should come from the CI and authors from git log

Reply:
ok, will remove it from ps meta then.